source/includes/fact-ansi-unicode-driver.rst

```diff
@@ -1,5 +1,6 @@
 .. note::
 
-   ANSI ODBC/Connectors offer maximum performance but have a limited
-   character set. Unicode ODBC/Connectors support a wider character
-   set but may be slightly less performant.
+   The ANSI ODBC driver (``libmdbodbca.so``) offers maximum
```

Comment:
Replace ODBC with `|odbc|`. It will provide an abbreviation definition.

Reply:
OK